app/controllers/application_controller.rb

```diff
@@ -50,12 +50,6 @@ def display_search_box_in_header?
 
   helper_method :display_search_box_in_header?
 
-  def display_menu_button_in_header?
-    true
-  end
-
-  helper_method :display_menu_button_in_header?
-
```

Comment:
[Yep](https://github.com/search?q=org%3Amoneyadviceservice+display_menu_button_in_header%3F&type=Code).

And `EmbeddedToolsController` was also one of the controllers that was overriding it for all tools.